gnomad_qc/v4/annotations/generate_freq_genomes.py

```diff
@@ -1485,7 +1516,7 @@ def main(args):
         )
         ht = get_histograms(ht, v3_sites_ht)
         ht = ht.annotate_globals(
```

Comment:
What about something like this?
```suggestion
        # NOTE: The v3.1 release HT doesn't have a downsamplings global. Since
        #  non-standard downsampling values are created in the frequency script
        #  corresponding to population totals, so this needs to be determined from the
        #  freq_meta.
        downsamplings = list(
            {
                str(x["downsampling"]) 
                for x in hl.eval(ht.freq_meta) if "downsampling" in x
            }
        )
        downsamplings.sort()
        
        ht = ht.annotate_globals(
            downsamplings=downsamplings,
```